rca/editorial/models.py

```diff
@@ -127,3 +128,100 @@ def get_context(self, request, *args, **kwargs):
         context["hero_image"] = self.hero_image
 
         return context
+
+
+class EditorialListingRelatedEditorialPage(RelatedPage):
+    source_page = ParentalKey(
+        "EditorialListingPage", related_name="related_editorial_pages"
+    )
+    panels = [PageChooserPanel("page", ["editorial.EditorialPage"])]
+
+
+class EditorialListingPage(BasePage):
+    template = "patterns/pages/editorial/editorial_listing.html"
+    subpage_types = ["editorial.EditorialPage"]
+
+    introduction = models.CharField(max_length=200, blank=True)
+
+    content_panels = BasePage.content_panels + [
+        FieldPanel("introduction"),
+        MultiFieldPanel(
+            [
+                InlinePanel(
+                    "related_editorial_pages", max_num=6, label="Editorial Pages"
+                ),
+            ],
+            heading="Editors picks",
+        ),
+    ]
+
+    def get_editor_picks(self, pages):
+        related_pages = []
+        for value in pages.select_related("page"):
+            if value.page and value.page.live:
+                page = value.page.specific
+
+                meta = None
+                if page.related_schools_and_research_pages.exists():
+                    meta = page.related_schools_and_research_pages.first().page.title
+
+                related_pages.append(
+                    {
+                        "title": page.title,
+                        "link": page.url,
+                        "image": page.listing_image or page.hero_image,
+                        "description": page.introduction
+                        if hasattr(page, "introduction")
```

Comment:
This will use `introduction` even if its blank, so `listing_summary` will never be used in preference (in case it needs to be)